chevah/compat/testing/assertion.py

```diff
@@ -0,0 +1,366 @@
+# -*- coding: utf-8 -*-
```

Comment:
even if we have unicode literals, we still need `# -*- coding: utf-8 -*-`

This diff is already big, and it was a bad idea to do these changes here... so I don't want to do more changes....

compat is already ported to py3 syntax

for compat we are not using unicode_literals, but the `future str`